exonum/src/runtime/rust/mod.rs

```diff
@@ -504,10 +504,14 @@ impl Runtime for RustRuntime {
         _snapshot: &dyn Snapshot,
         spec: &InstanceSpec,
         status: InstanceStatus,
-    ) -> Result<(), ExecutionError> {
+    ) {
         match status {
             InstanceStatus::Active => {
-                let instance = self.new_service(spec)?;
+                let instance = self
+                    .new_service(spec)
+                    .expect("BUG: Attempt to create a new service instance failed; \
+                             within `instantiate_adding_service` we were able to create a new instance,
```

Comment:
Nit: no `\` at the end of the line. Also, is this "hanging" style of formatting allowed by the formatter? I thought it would fix it to something like:

```rust
let instance = self
    .new_service(spec)
    .expect(
        "BUG: Attempt to create a new service instance failed; \
         within `instantiate_adding_service` we were able to create a new instance, \
         but now we are not.",
     );
```